python/cuml/cluster/dbscan.pyx

```diff
@@ -93,31 +93,25 @@ class DBSCAN(Base):
     Examples
     --------
 
-    .. code-block:: python
-
-            # Both import methods supported
-            from cuml import DBSCAN
-            from cuml.cluster import DBSCAN
-
-            import cudf
-            import numpy as np
-
-            gdf_float = cudf.DataFrame()
-            gdf_float['0'] = np.asarray([1.0,2.0,5.0], dtype = np.float32)
-            gdf_float['1'] = np.asarray([4.0,2.0,1.0], dtype = np.float32)
-            gdf_float['2'] = np.asarray([4.0,2.0,1.0], dtype = np.float32)
-
-            dbscan_float = DBSCAN(eps = 1.0, min_samples = 1)
-            dbscan_float.fit(gdf_float)
-            print(dbscan_float.labels_)
-
-    Output:
-
-    .. code-block:: python
-
-            0    0
-            1    1
-            2    2
+    >>> from cuml import DBSCAN
+    >>> from cuml.cluster import DBSCAN
+    >>>
+    >>> import cudf
```

Comment:
It looks like you are missing the leading comment `# Both import methods supported`

Reply:
```suggestion
    >>> # Both import methods supported
    >>> from cuml import DBSCAN
    >>> from cuml.cluster import DBSCAN
    >>>
    >>> import cudf
```
Adding suggestion so we can batch submit all comments at once.